apps/dev-proxy.config.js

```diff
@@ -1,15 +1,15 @@
-const HOST = process.env.MOLGENIS_APPS_HOST || "https://emx2.dev.molgenis.org";
+const HOST = process.env.MOLGENIS_APPS_HOST || "http://localhost:8080";
 const SCHEMA = process.env.MOLGENIS_APPS_SCHEMA || "pet%20store";
 
-const opts = { changeOrigin: true, secure: false }
+const opts = { changeOrigin: true, secure: false };
 
 module.exports = {
-  "^/graphql": {
```

Comment:
what issue does this solve , how can i test that this change fixes the issue ( if any)

Reply:
When I start with proxy to localhost:8080 I get following error with ^/ and not when /
![image](https://user-images.githubusercontent.com/120060/148781066-55f85fc2-3071-4a99-9188-b3284f76c367.png)